src/mdsynthesis/treants.py

```diff
@@ -2,8 +2,11 @@
 Basic Treant objects: the organizational units for :mod:`mdsynthesis`.
 
 """
+from __future__ import absolute_import
```

Comment:
What does `absolute_import` give us in this module?

Reply:
enforces that we need to do `from . import ...` for packae internal files under python2.7. It's a common idiom for python2/3 compatibility.